pkg/installer/deployresources.go

```diff
@@ -80,7 +80,7 @@ func zones(installConfig *installconfig.InstallConfig) (zones *[]string, err err
 	} else if zoneCount <= 2 {
 		zones = &installConfig.Config.ControlPlane.Platform.Azure.Zones
 	} else {
-		zones = &[]string{"[copyIndex(1)]"}
+		zones = &[]string{"[string(copyIndex(1))]"}
```

Comment:
Is casting as a string sufficient here?
If copyIndex does return an int, would we need to use `strconv.Itoa(copyIndex(1))`? I'm not sure how `strconv.Itoa()` handles a string being passed in, if/when it returns a string.

Reply:
Question answered, it's a call to an arm function.